k8s/cloud/base/indexer_deployment.yaml

```diff
@@ -14,6 +14,9 @@ spec:
       labels:
         name: indexer-server
         monitoring.gke.io/scrape: 'true'
+        prometheus.io/scrape: 'true'
+        prometheus.io/port: '51801'
+        prometheus.io/scheme: 'http'
```

Comment:
I tried to use a yaml anchor to avoid duplicating these, but skaffold wasn't compatible with it: